janusgraph-core/src/main/java/org/janusgraph/graphdb/types/ParameterType.java

```diff
@@ -35,8 +35,14 @@
     INDEX_GEO_MAX_LEVELS("index-geo-max-levels"),
 
     /** Distance error percent used to determine precision in spatial prefix tree where applicable. **/
-    INDEX_GEO_DIST_ERROR_PCT("index-geo-dist-error-pct");
-
+    INDEX_GEO_DIST_ERROR_PCT("index-geo-dist-error-pct"),
+    
+    /** Analyser for String Type with mapping STRING**/
+    STRING_ANALYZER("string-analyzer"),
+
+    /** Analyser for String Type with mapping TEXT**/
+    TEXT_ANALYZER("text-analyzer");
```

Comment:
semicolon on next line to reduce the number of diff lines next time we add to this Enum